### PR TITLE
[community] update PMC & Committer list

### PR DESCRIPTION
**Pull request by morningman**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/community/team.md [hunk 1]
```diff
@@ -24,62 +24,60 @@ specific language governing permissions and limitations
...
-| [wenming](https://people.apache.org/committer-index.html#wenming) | moonming        | Ming Wen          |
-| [shaofengshi](https://people.apache.org/committer-index.html#shaofengshi) | shaofengshi     | Shao Feng Shi     |
-| [ningjiang](https://people.apache.org/committer-index.html#ningjiang) | WillemJiang     | Willem Ning Jiang |
-
-We'd like to thank the following committers to the Apache Doris (incubating) project who have helped get the project to where it is today. This list might be stale, the canonical list is located on [Apache's website](https://people.apache.org/committers-by-project.html#doris).
-
-## PPMC (22)
-
-(the listing below excludes mentors)
+## PPMC (15)
```
Comment (kpfly) [inline docs/en/community/team.md:31]: PPMC or PMC?

File: docs/en/community/team.md [hunk 2]
```diff
@@ -24,62 +24,60 @@ specific language governing permissions and limitations
...
+| [sijie](https://people.apache.org/committer-index.html#sijie) | sijie               | Sijie Guo     |
+| [weixiang](https://people.apache.org/committer-index.html#weixiang) | spaces-X            | Xiang Wei     |
+| [zouxinyi](https://people.apache.org/committer-index.html#zouxinyi) | xinyiZzz               | Xinyi Zou     |
 | [wangxixu](https://people.apache.org/committer-index.html#wangxixu) | xinghuayu007     | Xixu Wang      |
 | [xuyang](https://people.apache.org/committer-index.html#xuyang) | xy720            | Yang Xu        |
 | [laiyingchun](https://people.apache.org/committer-index.html#laiyingchun) | acelyc111        | Yingchun Lai   |
+| [dataroaring](https://people.apache.org/committer-index.html#dataroaring) | dataroaring | Yongqiang Yang   |
 | [wyf](https://people.apache.org/committer-index.html#wyf)    | wuyunfeng        | Yunfeng Wu     |
+| [zenoyang](https://people.apache.org/committer-index.html#zenoyang)    |         | Zeno Yang     |
+| [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |
```
Comment (kpfly) [inline docs/en/community/team.md:79]: | [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |   
is this right

File: docs/zh-CN/community/team.md [hunk 3]
```diff
@@ -24,62 +24,60 @@ specific language governing permissions and limitations
...
+| [sijie](https://people.apache.org/committer-index.html#sijie) | sijie               | Sijie Guo     |
+| [weixiang](https://people.apache.org/committer-index.html#weixiang) | spaces-X            | Xiang Wei     |
+| [zouxinyi](https://people.apache.org/committer-index.html#zouxinyi) | xinyiZzz               | Xinyi Zou     |
 | [wangxixu](https://people.apache.org/committer-index.html#wangxixu) | xinghuayu007     | Xixu Wang      |
 | [xuyang](https://people.apache.org/committer-index.html#xuyang) | xy720            | Yang Xu        |
 | [laiyingchun](https://people.apache.org/committer-index.html#laiyingchun) | acelyc111        | Yingchun Lai   |
+| [dataroaring](https://people.apache.org/committer-index.html#dataroaring) | dataroaring | Yongqiang Yang   |
 | [wyf](https://people.apache.org/committer-index.html#wyf)    | wuyunfeng        | Yunfeng Wu     |
+| [zenoyang](https://people.apache.org/committer-index.html#zenoyang)    |         | Zeno Yang     |
+| [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |
```
Comment (kpfly) [inline docs/zh-CN/community/team.md:79]: | [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |

same here

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/en/community/team.md (hunk 1 above)
Comment (morningman) [reply docs/en/community/team.md]: fix

File: docs/en/community/team.md (hunk 2 above)
Comment (morningman) [reply docs/en/community/team.md]: fix

File: docs/zh-CN/community/team.md (hunk 3 above)
Comment (morningman) [reply docs/zh-CN/community/team.md]: fix

**Review by kpfly — APPROVED** [review]

**Review by caiconghui — APPROVED** [review]
LGTM

**Pull request merged**